packages/webpack/lib/utils/loader.js

```diff
@@ -15,25 +15,27 @@ const getMixins = (type, { _mixins: mixins }) => {
   return `[${requires.join(',')}]`;
 };
 
+const generateModuleCode = (type, config) => `
+${type === 'server' ? 'const { dirname, join } = require("path");' : ''}
```

Comment:
A single condition would be enough here imho.

```diff
- ${type === 'server' ? 'const { dirname, join } = require("path");' : ''}
- ${type === 'server' ? 'const { sync: findUp } = require("find-up");' : ''}
- ${type === 'server' ? 'const rootDir = dirname(findUp("package.json"));' : ''}
- ${type === 'server' ? 'const expand = join.bind(null, rootDir);' : ''}
+ ${type === 'server' ? `
+   const { dirname, join } = require("path");
+   const { sync: findUp } = require("find-up");
+   const rootDir = dirname(findUp("package.json"));
+   const expand = join.bind(null, rootDir);
+ ` : ''}
```

Reply:
Yes, I know - probably another matter of personal/aesthetic preference, I guess. I think an additional `const` might do the trick...

Reply:
Done - made it another helper function (technically also a `const`).